sharktank/sharktank/examples/paged_llm_v1.py

```diff
@@ -236,12 +237,12 @@ def main():
         activation_dtype=activation_dtype,
         attention_dtype=activation_dtype,
     )
-    model = PagedLlamaModelV1(dataset.root_theta, config)
-    if args.save_intermediates_path:
```

Comment:
We would want to retain this `if ` block, it's present in the main branch.

Reply:
great catch, thank you